### ISSUE TITLE
Add RDFS and OWL reasoning to RDF graph

### ISSUE THREAD

**Issue by mwatts15** [openworm/PyOpenWorm#26]
Some of the queries we perform in PyOpenWorm could be simplified by including a reasoner like [FuXi](https://github.com/RDFLib/FuXi). For instance, we currently insert an `rdf:type` triple for each super-class of a DataObject's type in order to utilize classes in our queries.

It would be preferable to augment the RDF graph with RDFS reasoning to infer that, for instance, a Neuron is a Cell, without inserting extra triples beyond a description of the class hiearchy (i.e., a rdfs:subClassOf link between Neuron and Cell). While it is be possible to modify individual queries to search the class hierarchy, by pushing this logic into a reasoner, we retain the simplified structure of our object -> triples -> query path in all present and future queries added to PyOpenWorm as well as custom queries devised by users of the library.

FuXi was chosen as a possible reasoner because it already interfaces with RDFLib.

See the [old Fuxi docs](https://code.google.com/p/fuxi/wiki/Tutorial#Programmatic_Equivalent) for a quick look at how to interface FuXi with a graph.

Changes should be made in PyOpenWorm/data.py by replacing the value that goes into rdf.graph. See [here](https://github.com/openworm/PyOpenWorm/blob/alpha0.5/PyOpenWorm/data.py#L219). The graph can also be set directly. After calling PyOpenWorm.connect():

```
PyOpenWorm.Configureable.default['rdf.graph'] = <your rdflib graph>
```

**Comment by travs** [comment]
This is related to #176 and is fixed by #205

**Comment by slarson** [comment]
Closed by #206
